src/main/cpp/chassis/SwerveChassis.cpp

```diff
@@ -45,6 +45,9 @@
 
 // Third Party Includes
 #include "pugixml/pugixml.hpp"
+#include <ctre/phoenix6/StatusSignal.hpp>
+
```

Comment:
Did you mean to keep the timer and logging in?

Reply:
removed